about.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About</title>
+  <link rel="stylesheet" href="aboutStyle.css">
+  <!-- CSS only -->
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-gH2yIJqKdNHPEq0n4Mqa/HGKIhSkIHeL5AyhkYV8i59U5AR6csBvApHHNl/vI1Bx" crossorigin="anonymous">
+  <link rel="stylesheet"
+    href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+
+</head>
+
+<body>
+
+  <header class="position-sticky top-0">
+    <div class="nav-1 bg-dark">
+      <ul class="d-flex float-right gap-3 m-0 justify-content-end p-3 align-items-center">
+        <li class="text-white "><i class="fa-brands fa-square-facebook"></i> </li>
+        <li class="text-white "><i class="fa-brands fa-github"></i></li>
+        <li class="text-white">Logout</li>
+        <li class="text-white">Mypage</li>
+        <li class="text-white">India</li>
+      </ul>
+    </div>
+    <nav class="navbar navbar-expand-md  bg-white">
+      <div class="container toggles justify-content-md-evenly">
+        <a class="navbar-brand" href="index.html"><i class="fa-solid fa-won-sign fa-2xl"></i></a>
+        <button class=" btn burger d-md-none" type="button"><i class="fa-solid fa-bars"></i></button>
+        <button type="button" class="btn d-none close-btn"><i class="fa-solid fa-xmark "></i></button>
+        <div class="justify-content-evenly d-none d-md-flex" id="navbarNav">
+          <ul class="navbar-nav">
+            <li class="nav-item"><a class="nav-link " aria-current="page" href="about.html">About</a>
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you change this `<a href>` to link to home page instead of linking back to itself.

## EXAMPLE
`<li><a class="nav-link " aria-current="page" href="./index.html">Home</a></li>`